docker/entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+    local var="$1"
+    local fileVar="${var}_FILE"
+    local def="${2:-}"
+    if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
+        echo >&2 "error: both $var and $fileVar are set (but are exclusive)"
+        exit 1
+    fi
+    local val="$def"
+    if [ "${!var:-}" ]; then
+        val="${!var}"
+    elif [ "${!fileVar:-}" ]; then
+        val="$(< "${!fileVar}")"
+    fi
+    export "$var"="$val"
+    unset "$fileVar"
+}
+
+file_env 'IOTA_CB_HOST'
+file_env 'IOTA_CB_PORT'
+file_env 'IOTA_MONGO_HOST'
+file_env 'IOTA_MONGO_PORT'
+file_env 'IOTA_AUTH_USER'
+file_env 'IOTA_AUTH_PASSWORD'
+
+if [[ $IOTA_AUTH_USER ==  "iotagent" ]] || [[ $IOTA_AUTH_PASSWORD == "iotagent'" ]]; then
+        echo "***********************************************"
+        echo "WARNING: It is recommended to change IOTA Auth credentials not to use default values"
```

Comment:
This statement is wrong - without an override, the IoT Agent will fail to connect - it should be an ERROR

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/df770138c08a0de85d4a79a673af0d1477ec4d8e